src/ray/object_manager/plasma/test/fallback_allocator_test.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <boost/filesystem.hpp>
+#include "gtest/gtest.h"
+#include "ray/object_manager/plasma/plasma_allocator.h"
+
+using namespace boost::filesystem;
+
+namespace plasma {
+namespace {
+const int64_t kMB = 1024 * 1024;
+std::string CreateTestDir() {
+  path directory = temp_directory_path() / unique_path();
+  create_directories(directory);
```

Comment:
Btw, all these mmaped files destroyed when the allocator is GC'ed? Do you know when that happens?

Reply:
not the first one for some reason...

Reply:
Hmm wonder if fallback allocated filed are going to take up disk space after allocator is destructed… if so we might need some mechanism to clean up?

Reply:
(But it can be todo if it is not regression)

Reply:
oh, fallback allocated file are always reclaimed, so there is no concern. only the /dev/shm not reclaimed.